testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveRedirectionToIceberg.java

```diff
@@ -476,6 +476,21 @@ public void testGrant()
         onTrino().executeQuery("DROP TABLE " + icebergTableName);
     }
 
+    @Test(groups = {HIVE_ICEBERG_REDIRECTIONS, PROFILE_SPECIFIC_TESTS})
+    public void testRevoke()
```

Comment:
We should have also a test in `io.trino.security.TestAccessControl`

Reply:
The `ServerMainModule` uses by default `DisabledSystemSecurityMetadata` which doesn't have support for granting, revoking, denying permissions.

The exception message logged from the test:

```
Catalog does not support permission management: blackhole
```